h/static/styles/partials-v2/_search.scss

```diff
@@ -0,0 +1,91 @@
+// Search page
+// -----------
+// Styles for the search page which displays the results
+// of activity queries.
+
+.search {
+  font-size: $normal-font-size;
+  margin-top: 20px;
+  color: $grey-6;
+}
+
+.search-results {
+  list-style: none;
+  width: 950px;
+}
+
+.search-result {
```

Comment:
I would probably change the naming here slightly because this component doesn't refer to a single search result (which would be one annotation card) but a _bucket_ of search results (to use the terminology used in the Python code) which all belong to the same document and were all updated in the same time frame. `search-result-bucket` perhaps?

There was a lengthy discussion on Slack a while back about how search is going to work. Conceptually the important thing that we (and users) need to understand is that it searches annotations and then groups/"buckets" the result by document and last-update time.